### DISCUSSION TITLE
Darker mouth if teeth are shown

### DISCUSSION THREAD

**Discussion by Madis0** [C1710/blobmoji#67] [Style]
I've noticed that in small fonts, the mouths aren't as legible as in other emoji sets.

For example, the widely used [grinning face](https://emojipedia.org/grinning-face/) 😀 may seem like it is more about the white teeth, rather than the smile itself.

Similarly, [the tears of joy](https://emojipedia.org/face-with-tears-of-joy/) 😂  slightly look like fangs due to lack of contrast.

Which dark color to use, though, is up to you. I'd prefer red (Samsung) or black/gray (Google, EmojiOne, Microsoft), not yellow (Apple, WhatsApp, Twitter).

Edit: Not all mouths need to be changed, only those that show teeth.

**Comment by C1710** [comment]
Erm, somehow I missed this one...  
I am still trying to figure out a better way to show the emojis with teeth - not only the color (which I personally haven't experienced any problems with) - but rather with their overall style. I don't really like this white band as it is now, but I don't know any alternative yet...

**Comment by C1710** [comment]
Is it better now?
https://github.com/C1710/blobmoji/blob/emoji13/svg/emoji_u1f600.svg

**Reply by Madis0** [reply]
Hmm, looks like it still needs more contrast.

**Reply by C1710** [reply]
> 
> 
> Hmm, looks like it still needs more contrast.

This is already the darkest orange in the palette. I have now [added 10% black](https://github.com/C1710/blobmoji/blob/emoji13-1/svg/emoji_u1f600.svg), but I wouldn't go any darker as it doesn't fit in with the other emojis anymore.

Red or black isn't really an option as it would be very inconsistent with the other blobs.
